pymc3/tests/test_distributions.py

```diff
@@ -2573,10 +2571,6 @@ def test_logitnormal(self):
             decimal=select_by_precision(float64=6, float32=1),
         )
 
-    @pytest.mark.xfail(
```

Comment:
These logp/logcdf tests probably have to stay as `xfail`. The problem is that only a subset of parameter / values is tested on each rain and the failing cases don't show up always.

Reply:
But that doesn't work in combination with the strict flag.
Which parameter values are the problematic ones?
Or what's the exception and then we'll catch that specifically.

Reply:
I undid the removal of the xfail mark, but added an Exception raised on purpose to prevent XPASS.